File: app/src/main/java/alejandromr92/com/movies/utils/LoggerUtils.java
package alejandromr92.com.movies.utils;

import android.util.Log;

public abstract class LoggerUtils {

    public static void logMessage(String tag, String message){
        Log.d(tag, message);
    }

    public static void logError(String tag, String message, Exception exception){
        Log.e(tag, message, exception);
    }
}

File: app/src/main/java/alejandromr92/com/movies/utils/LoggerUtils.kt
package alejandromr92.com.movies.utils

import android.util.Log

/**
 * Helper class to register logs.
 */
object LoggerUtils {

    fun logMessage(tag: String, message: String) {
        Log.d(tag, message)
    }

    fun logError(tag: String, message: String, exception: Exception) {
        Log.e(tag, message, exception)
    }
}
